doc/en/_static/pytest.css

```diff
@@ -0,0 +1,8 @@
+/* Hide list bullets and top-level indent in TOC sidebar */
+.sidebar-scroll ul {list-style: none;}
+.sidebar-scroll > ul {padding-left: 0;}
+.sidebar-scroll li {margin: 0.4em 0;}
```

Comment:
A slight change from @pradyunsg's #12575, as I discovered that the navigation sidebar _isn't_ styled `.sidebar` (surprisingly), but the events sidebar **is**. So, nesting these changes under `.sidebar` would've hit the wrong part of the docs.

The navigation content-list is under a div at `.sidebar-drawer > .sidebar-container > .sidebar-scroll`.

I suppose I could've used either of those parent selectors instead, except for the `.sidebar-scroll > ul` change.

Reply:
Wait, this was not meant for TOC sidebar at all. It's meant for the literal "sidebar" thing in reStructuredText: https://pradyunsg.me/furo/kitchen-sink/generic/#sidebar

Reply:
My understanding is that folks didn't like the extra indentation that having list items gave their custom sidebar.

Reply:
Oh, no, pytest devs _want_ list markers in the event sidebar. The original CSS this replaces was intended to remove them only from the TOC/navigation sidebar, but it was overzealous by a lot.

Reply:
Oh, fun.